src/Humanizer.Tests.Shared/CollectionHumanizeTests.cs

```diff
@@ -87,5 +88,58 @@ public void HumanizeUsesObjectFormatterWhenSeparatorIsProvided()
             var humanized = _testCollection.Humanize(sc => string.Format("SomeObject #{0} - {1}", sc.SomeInt, sc.SomeString), "or");
             Assert.Equal("SomeObject #1 - One, SomeObject #2 - Two, or SomeObject #3 - Three", humanized);
         }
+
+        [Fact]
+        public void HumanizeHandlesNullItemsWithoutAnException()
+        {
+            new object[] { null, null }.Humanize();
```

Comment:
Ok. Record.Exception instead of Asser.DoesNotThrow then. I don't care how
it will be implemented, but what I care - is explicitly, that this
particular call to Humanize does not throw
On Tue, 28 Jun 2016 at 9:39 AM, Joseph Musser <notifications@github.com>
wrote:

> In src/Humanizer.Tests.Shared/CollectionHumanizeTests.cs
> <https://github.com/Humanizr/Humanizer/pull/566#discussion_r68661345>:
>
> > @@ -87,5 +88,58 @@ public void HumanizeUsesObjectFormatterWhenSeparatorIsProvided()
> >              var humanized = _testCollection.Humanize(sc => string.Format("SomeObject #{0} - {1}", sc.SomeInt, sc.SomeString), "or");
> >              Assert.Equal("SomeObject #1 - One, SomeObject #2 - Two, or SomeObject #3 - Three", humanized);
> >          }
> > +
> > +        [Fact]
> > +        public void HumanizeHandlesNullItemsWithoutAnException()
> > +        {
> > +            new object[] { null, null }.Humanize();
>
> Sure, but I find DoesNotThrow redundant and not in a nice way. The whole
> premise of its existence flies in the face of how the language itself works.
>
> It's on track to be removed from xunit, and I have to say I agree with the
> sentiments there: xunit/xunit#188
> <https://github.com/xunit/xunit/issues/188>
>
> Other projects like dotnet/roslyn have been removing all usages of
> DoesNotThrow. I'd hate to perpetuate it. Do I have to?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Humanizr/Humanizer/pull/566/files/7abd8e2ffe497ad51598278bd8b0c8130f928512#r68661345>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAI1l8HPS53NAPeZ972HBsFt75gKvNeXks5qQEMegaJpZM4I7H4x>
> .
>